CCC-TAC_Meeting_Minutes-2025-01-23 (#269)
confidential-computing/governance: TAC/Meetings/2025/2025-01-23/CCC-TAC-Minutes-2025-01-23.md

DM asked about the MS Open Source Code of Conduct. CPV shall look into the differences between the Contributor Covenent and the Microsoft code of conduct 

DM asked about copy left implications. CPV said they do not foresee implications.

MB asked about the existing community and how the project would extend the community and if there are other contributors then MS? CPV the project already had external contributions, however most is still done by MS. They are all looking at ways to socialize the project. Looking for more partners and having public meetings. They are also looking for guidance from the CCC on how to reach these goals.

DM ask if their is a way for contributors to become maintainers and if the project are interested in external maintainers. CPV said they plan for it, but would develop the plan with the community rather than use their own internal arrangements.

The motion to accept the project proposal was made by Alec Fernandez and seconded by  Paul Howard.

* Paul Howard - ARM - Yes
* Intel - Dan Middleton (TAC chair) - Yes
* Microsoft - Alec Fernandez - Yes 
* Red Hat - Yash Mankad - Yes
* Fritz Alder (NVIDIA) - Yes

> The project has been approved by the TAC. The voting members that were not in the TAC meeting would also be offered the opportunity to vote by email. The project proposal will be taken to the Governing Board meeting as well for approval.

Reviewer: ```suggestion
The project has been approved by the TAC. The voting members that were not present will also be offered the opportunity to vote by email. The project proposal will be taken to the Governing Board meeting as well for approval.
```
Please add to the minutes the email votes of the absent members (noting that they were sent by email).
